app/services/find/match_old_params.rb

```diff
@@ -0,0 +1,78 @@
+module Find
+  class MatchOldParams
+    include ServicePattern
+
+    FILTERS = {
+      "senCourses" => "send_courses",
+      "lat" => "latitude",
+      "lng" => "longitude",
+      "rad" => "radius",
+      "query" => "provider.provider_name",
+      "hasvacancies" => "has_vacancies",
+      "subject_codes" => "subjects",
+    }.freeze
+
+    STUDY_FILTERS = {
+      "parttime" => "part_time",
+      "fulltime" => "full_time",
+    }.freeze
+
+    QAULIFICATION_FILTERS = {
+      "Other" => "pgce pgde",
+      "PgdePgceWithQts" => "pgce_with_qts",
+      "QtsOnly" => "qts",
+    }.freeze
+
+    def initialize(request_params)
+      @request_params = request_params
+    end
+
+    def call
+      sortby_and_funding
+      qualification_filters
+      filters
+      study_filters
+
+      @request_params
+    end
+
+  private
+
+    def sortby_and_funding
+      @request_params["sortby"] = "distance" if @request_params["sortby"] == "2"
+      @request_params["funding"] = "salary" if @request_params["funding"] == "8"
+    end
+
+    def qualification_filters
+      if @request_params["qualifications"]
+        @request_params["qualification"] = @request_params.delete("qualifications")
```

Comment:
:woozy_face:

Reply:
I think that is correct?

Reply:
I not sure what you are intending....

![image](https://user-images.githubusercontent.com/470137/213675676-dd0d0747-7c57-403c-918e-72906c8382b6.png)

Reply:
x = Hash.new
x[:qualifications] = "awesome"
x[:qualification] = x.delete(:qualifications)

3.1.0 :006 > x
 => {:qualification=>"awesome"}

Reply:
So in the test 
```
context "with QUALIFICATION_FILTERS" do
      subject {
        described_class.call({
          "qualifications" => %w[Other PgdePgceWithQts QtsOnly],
        })
      }

      it "maps the old find params" do
        expect(subject).to eq({
          "qualification" => ["pgce pgde", "pgce_with_qts", "qts"],
```

Reply:
Its 'qualificationS' mapped to 'qualification'

Reply:
:woozy_face: